desktop.ini

```diff
@@ -0,0 +1,6 @@
+[.ShellClassInfo]
```

Comment:
Why do you need this file?

Reply:
I didn't modify the folder desktop.ini so I am not sure how I ended up doing this!